### PR TITLE
added some debug messages

### PR DESCRIPTION
**Pull request by pnedelko**

### CODE REVIEW

File: lib/service.js [hunk 1]
```diff
@@ -34,12 +35,12 @@ function processService(serviceMethod, env){
         .catch(onError);
 
     function init() {
-
         assert(ALLOWED_METHODS[serviceMethod], util.format('Unknown service method "%s"', serviceMethod));
```
Comment (drobiazko) [inline lib/service.js]: Maybe one more here as well? For example: ``About to init``

File: lib/service.js [hunk 2]
```diff
@@ -64,7 +67,10 @@ function processService(serviceMethod, env){
     }
 
     function execService(cfg, params) {
+        debug('Init is complete');
```
Comment (drobiazko) [inline lib/service.js]: About to start execution

File: lib/service.js [hunk 3]
```diff
@@ -81,12 +87,11 @@ function processService(serviceMethod, env){
...
             message: err.message
         };
-        console.error(err.stack);
+        console.error(err, err.stack);
         return sendResponse({status: 'error', data: errorData});
     }
 
-    function sendResponse(responseBody){
-        assert(POST_RESULT_URL, 'ELASTICIO_POST_RESULT_URL is not provided');
+    function sendResponse(responseBody) {
```
Comment (drobiazko) [inline lib/service.js]: About to send response to admiral

Comment (pnedelko) [inline lib/service.js]: added on line 104

**Pull request merged**